src/index.css

```diff
@@ -13,6 +13,15 @@
     --font-heading-2: var(--font-heading);
     --font-heading-3: var(--font-heading);
 
+    /* Customizable font categories - default to existing fonts for backward compatibility */
+    --font-heading-1: var(--font-heading);
+    --font-heading-2-custom: var(--font-heading);
+    --font-heading-3-custom: var(--font-heading);
```

Comment:
Don't add new variables for font-heading, font-heading-2, and font-heading-3 "custom" - just use the original ones.